CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - ...
 
+## [v1.0.0] - 2020-12-24
```

Comment:
My only concern with a version 1 is whether or not it's ready for "production".  Otherwise, I think it's fine.

Care to share how you use this package?  I have confidence in how I use it, but I wonder how others use it.  🤓

Reply:
We use it to declare Structural units tree inside a large organization.